packages/icon-button/README.md

```diff
@@ -57,8 +56,11 @@ import IconButton from '@leafygreen-ui/icon-button';
 | `href`      | `string`                     | If a href is supplied, the component renders inside of an `a` tag instead of inside of a `button` tag.            |           |
 | `className` | `string`                     | Adds a className to the class attribute on the container element.                                                 |           |
 | `children`  | `node`                       | Content rendered inside of the `<IconButton />` component                                                         |           |
-| `ariaLabel` | **Required** `string`        | Value passed to `aria-label` attribute to ensure component's accessibility                                        |           |
 | `size`      | `default`, `large`, `xlarge` | Determines the size of the IconButton                                                                             | `default` |
 | `active`    | `boolean`                    | Determines whether the `<IconButton />` will appear `active`                                                      | `false`   |
 
+### Special Case: Aria Labels
```

Comment:
We have required props across multiple components, why break this out into its own section here?

Reply:
I broke it out because in the table, it's particularly difficult to indicate that having one of the two is required, but they're both optional. It's a little strange compared to our usual prop typing.

Reply:
Kk that makes sense